frontend/src/component/application/ApplicationList/ApplicationList.tsx

```diff
@@ -100,15 +101,8 @@ export const ApplicationList = () => {
                 Header: 'Project(environment)',
                 accessor: 'usage',
                 width: '50%',
-                Cell: () => (
-                    <TextCell>
-                        <Typography
-                            variant="body2"
-                            color={theme.palette.text.secondary}
-                        >
-                            not connected
-                        </Typography>
-                    </TextCell>
+                Cell: ({ row: { original } }: any) => (
```

Comment:
can we write explicit type here to avoid guessing/runtime inspection?

Reply:
Added type